.editorconfig

```diff
@@ -2,18 +2,10 @@ root = true
 
 [*]
 indent_style = space
-indent_size = 2
```

Comment:
Can you keep the updates minimal? i.e. only end of line? html should be kept at indent_size 4.